L'exécution de votre programme a déclenché une exception java.lang.RuntimeException : Désolé Dave, je ne peux pas te laisser utiliser setPos(x,y) dans cet exercice. Marche jusqu'à ton objectif.
 Merci de corriger votre code.
   at plm.runtime40.JediEscaper.setPos (plm.runtime40.JediEscaper:19)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1$$anonfun$apply$mcVI$sp$3.apply$mcVI$sp (plm.runtime40.JediEscaper:78)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1$$anonfun$apply$mcVI$sp$3.apply (plm.runtime40.JediEscaper:73)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1$$anonfun$apply$mcVI$sp$3.apply (plm.runtime40.JediEscaper:73)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1.apply$mcVI$sp (plm.runtime40.JediEscaper:73)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1.apply (plm.runtime40.JediEscaper:71)
   at plm.runtime40.JediEscaper$$anonfun$repererSortie$1.apply (plm.runtime40.JediEscaper:71)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime40.JediEscaper.repererSortie (plm.runtime40.JediEscaper:71)
   at plm.runtime40.JediEscaper.run (plm.runtime40.JediEscaper:38)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (null:-1)
